packages/pool/contracts/DelegationUtils.sol

```diff
@@ -22,6 +25,7 @@ abstract contract DelegationUtils is RewardUtils, IDelegationUtils {
                 && userDelegate(delegate) == address(0),
             ERROR_ADDRESS
             );
+        require(userShares(msg.sender) > proposalVotingPowerThreshold, ERROR_DELEGATION_BALANCE );
```

Comment:
`userShares` must be greater than `0`, not `proposalVotingPowerThreshold`

Reply:
Also `userShares()` is already being called below, you're calling it twice now. Use the `userShares` calculated below.

Reply:
Ok, but userShares for msg.sender should be more then proposalVotingPowerThreshold to submit proposals. This way we will kind-of limit the issue with 20 voting delegations, because they will send at least 2% of the voting power to address they want to block of receiving proposals...

Reply:
Anyway, if you think we need change it, I will

Reply:
Yeah, the whole point of delegating is that you have very little voting power and it's not worth it for you to vote for yourself. Putting a threshold for delegation defeats its purpose. I'll address the 20 proposals thing separately.